lib/config/config.go

```diff
@@ -39,13 +39,16 @@ const (
 var (
 	// DefaultTCPPort defines default TCP port used if the URI does not specify one, for example tcp://0.0.0.0
 	DefaultTCPPort = 22000
+	// DefaultQUICPort defines default QUIC port used if the URI does not specify one, for example quic://0.0.0.0
+	DefaultQUICPort = 22442
```

Comment:
I really feel we should use 22000 here as well as it's sort of mentally ingrained as the Syncthing port. I forget what the reason was to do something else? (But I remember we did talk about this before.)